package.json

```diff
@@ -21,10 +21,10 @@
   ],
   "scripts": {
     "clean": "rimraf .tap node_modules types",
-    "test": "cross-env HTTP_PORT=0 LOG_LEVEL=fatal tap --timeout 0 --disable-coverage",
-    "test:integration": "cross-env HTTP_PORT=0 LOG_LEVEL=fatal tap --timeout 0 --disable-coverage test/integration/**/*.test.mjs",
-    "test:unit": "cross-env HTTP_PORT=0 LOG_LEVEL=fatal tap --timeout 0 --disable-coverage test/*.test.mjs",
-    "test:tasks": "cross-env HTTP_PORT=0 LOG_LEVEL=fatal tap --timeout 0 --disable-coverage test/tasks/*.test.mjs",
+    "test": "cross-env HTTP_PORT=0 LOG_LEVEL=fatal tap --timeout 60 --disable-coverage",
```

Comment:
what was the reason for changing to timeout 60?

Reply:
I didn't get error output from the `beforeEach` hook until the test was finished, but it would never finish since the server didn't close properly and there was an infinite timeout.